### DISCUSSION TITLE
Awaiting Github Student Benefits

### DISCUSSION THREAD

**Discussion by theomarentes** [community/community#60537] [GitHub Education]
### Hi everyone,

I have recently been approved into the Github student program, and I am eagerly waiting on my benefits to arrive.
I got approved on the 8th of July, so it has been close to 72 hours.
How long does it usually take to get approved?

Thanks
Theo

**Comment by STOL4S** [comment]
Longer than 7 days

**Comment by samus-aran** [comment]
Hey @theomarentes  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
